joern-cli/frontends/javasrc2cpg/src/main/scala/io/joern/javasrc2cpg/typesolvers/noncaching/JdkJarTypeSolver.scala

```diff
@@ -134,7 +135,8 @@ object JdkJarTypeSolver {
   }
 
   def fromJdkPath(jdkPath: String): JdkJarTypeSolver = {
-    val jarPaths = SourceFiles.determine(jdkPath, Set(JarExtension, JmodExtension))
+    // not following symlinks, because some setups might have a loop, e.g. AWS's Corretto
```

Comment:
```suggestion
    // not following symlinks, because some setups might have a loop, e.g. AWS's Corretto
    // see https://github.com/joernio/joern/pull/3871
```